_episodes/04-use-license.md

```diff
@@ -5,7 +5,185 @@ exercises: 0
 questions:
 - "Key question"
 objectives:
-- "First objective."
+- Tell what is a copyright and what a licence does
+- tell why is important that a product/code has a licence
+- tell what is the importance of third-party dependencies on your product/code
```

Comment:
Regarding the licenses... maybe not necessary to add this clarification here are this is the episode about licenses, up to you.